### PR TITLE
Add embed colors and make use of HTTP status codes

### PR DESCRIPTION
**Pull request by ShockTr**
This PR adds colors to embeds for a nicer look and makes use of HTTP codes to make Crowdin detect unsuccessful webhook requests 

Embed colors

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -20,6 +20,12 @@ const mappers: Record<CrowdinEvents, (num: number) => string> = {
   },
 }
 
+const colors: {[event: string]:number} = {
```
Comment (Xetera) [inline src/index.ts:23]: This part should be properly typed
```suggestion
const colors: Record<CrowdinEvent, number> = {
```

Comment (ShockTr) [reply src/index.ts]: Fixed it

**Comment by Pacmard** [comment]
Lgtm

**Review by Xetera — APPROVED** [review]
Looks fine now. We probably need to add validation for the events we allow so we can safely cast things to `CrowdinEvents[]` later

**Comment by bashcoders** [comment]
simple join server https://discord.gg/vsHzWPwq :)

**Comment by bashcoders** [comment]
yuyiuyiyuiuyi

**Comment by bashcoders** [comment]
iyuiyuiuyi

**Comment by bashcoders** [comment]
yuiyuiuyiuyi

**Comment by bashcoders** [comment]
yuiyuiyuiyuiyui

**Comment by bashcoders** [comment]
yuiyuiyuiuyiyui

**Comment by bashcoders** [comment]
yuiyuiyuiyui

**Comment by bashcoders** [comment]
yuiuyiyuiyuiyi

**Comment by bashcoders** [comment]
yiyuiyuiyui

**Comment by bashcoders** [comment]
yuiyuiyuiyuiyui

**Comment by bashcoders** [comment]
yuiyuiyuiyuiyui

**Comment by bashcoders** [comment]
uyiyuiyuiyuiyuiyui

**Comment by bashcoders** [comment]
yiyuiyuiyuiyuiyui

**Comment by bashcoders** [comment]
iyuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuu

**Comment by bashcoders** [comment]
https://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwq

**Comment by bashcoders** [comment]
https://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwq

**Comment by bashcoders** [comment]
https://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwq

**Comment by bashcoders** [comment]
https://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwqhttps://discord.gg/vsHzWPwq

**Comment by bashcoders** [comment]
ههههه

**Comment by bashcoders** [comment]
ههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههههه

**Pull request merged**